src/components/wizard/wizard.component.ts

```diff
@@ -200,13 +202,15 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
       * Whether the Next or Finish button should be disabled.
       */
     isNextDisabled(): boolean {
-
+        // Check if there is a step
+        if (this.step > 0) {
         const step = this.getCurrentStep();
```

Comment:
I think the safer fix would be to check `step` here and return false if it's undefined, e.g.
```ts
if (!step) {
  return false;
}
```